cordformation/src/test/kotlin/net/corda/plugins/DockerformTest.kt

```diff
@@ -132,5 +134,24 @@ class DockerformTest : BaseformTest() {
         assertTrue(bankOfCordaConfig.hasPath("rpcSettings.adminAddress"))
         assertEquals(10006, ConfigurationUtils.parsePort(bankOfCordaConfig.getString("rpcSettings.address")))
         assertEquals(10046, ConfigurationUtils.parsePort(bankOfCordaConfig.getString("rpcSettings.adminAddress")))
+
+        val dockerComposePath = getDockerCompose()
+
+        val yaml = Yaml().load(dockerComposePath.toFile().bufferedReader()) as Map<Any, Any>
+        assertThat(yaml.containsKey("services")).isTrue()
+
+        val services = yaml["services"] as Map<Any, Any>
+        assertThat(services.containsKey("notary-service")).isTrue()
+        assertThat(services.containsKey("bankofcorda")).isTrue()
```

Comment:
AssertJ is very flexible in its assertions, and it understands both maps and lists. Hence:
```kotlin
assertThat(services).containsKey("notary-service")
```